.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-https://github.com/nimblehq/react-templates/issues/??
+Closes https://github.com/nimblehq/react-templates/issues/??
```

Comment:
```suggestion
- Close #
```

So you just need to add the issue number 🤩 

PS: from https://github.com/nimblehq/git-template/blob/main/.github/PULL_REQUEST_TEMPLATE.md?plain=1 ;-)

Reply:
Updated in d797a52e2511958f1e75cec25b736ad05f25a0f9 🪄